apps/wing-playground/index.html

```diff
@@ -59,6 +81,7 @@ <h1>Wing Playground</h1>
     <select id="example"></select>
     <main>
       <section id="wing-side">
+        <textarea name="lineCount" id="lineCounter" wrap="off" readonly></textarea>
         <textarea id="code"></textarea>
```

Comment:
There is still a bug with wrapping, so it needs to be disabled for the code the line numbers to always match.

I would suggest adding CSS for this element:

```css
white-space: nowrap
```

Reply:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/1237390/212965835-833df67b-1a45-45ee-8eac-1fad6ef36428.png">

I meant adding nowrap on the wing code, not the line-numbers. The logic for line numbers doesn't work when the wing code wraps

Reply:
done